types/FluentJSONSchema.d.ts

```diff
@@ -1,172 +1,169 @@
-export interface BaseSchema<T> {
-  id: (id: string) => T
-  title: (title: string) => T
-  description: (description: string) => T
-  examples: (examples: Array<any>) => T
-  ref: (ref: string) => T
-  enum: (values: Array<any>) => T
-  const: (value: any) => T
-  default: (value: any) => T
-  required: (fields?: string[]) => T
-  ifThen: (ifClause: JSONSchema, thenClause: JSONSchema) => T
+interface BaseSchema<T> {
```

Comment:
Exported `interface` should remain be exported inside `namespace`

Reply:
please see last commit

Reply:
This is not the only one. You need to update the others.

Reply:
updated: all interfaces are now inside namespace